packages/rfw/test/runtime_test.dart

```diff
@@ -347,6 +347,17 @@ void main() {
     expect(tester.widget<ColoredBox>(find.byType(ColoredBox)).color, const Color(0x00000002));
   });
 
+  testWidgets('Runtime', (WidgetTester tester) async {
+    final Runtime runtime = Runtime()
+          ..update(const LibraryName(<String>['core']), createCoreWidgets());
+    for (final MapEntry<LibraryName, WidgetLibrary>(key: LibraryName libraryName, value: WidgetLibrary widgetLibrary) in runtime.libraries.entries) {
+      expect('$libraryName', 'core');
+      expect(widgetLibrary, isA<LocalWidgetLibrary>());
+      final LocalWidgetLibrary localWidgetLibrary = widgetLibrary as LocalWidgetLibrary;
```

Comment:
(fwiw I believe you don't need the variable here, you can just say `widgetLibrary as LocalWidgetLibrary;` and then use `widgetLibrary` as the new type, due to promotion. Either is fine though.)

Reply:
Interesting, Dart has changed a lot in the last 7 years.

Reply:
Done.